services/kubecost/metadata.yaml

```
@@ -30,7 +30,7 @@ overview: |-
 
   ## More Information
   - [kubecost.com](https://www.kubecost.com/)
-  - [Centralized Cost Monitoring](https://docs.d2iq.com/dkp/kommander/latest/monitoring/centralized-cost-monitoring/)
+  - [Centralized Cost Monitoring](https://docs.d2iq.com/dkp/2.3/centralized-cost-monitoring)
```

```suggestion
  - [Centralized Cost Monitoring](https://docs.d2iq.com/dkp/latest/centralized-cost-monitoring)
```
or `2.4` perhaps - cuz `main` is ahead of `2.3` ?